### ISSUE TITLE
Analog Clock Functionality

### ISSUE THREAD

**Issue by klattimer** [klattimer/Teasmade#1]
A very specific set of frequencies are required to drive the clock mechanism properly. Somewhere in the 33khz range should be perfect. But driving it faster or slower than that causes problems. Need to measure the dilation of time based on a set of fixed constants so I can then work out how to get the facetime accurate... And of course, make the hands spin round wildly via a HTTP command in order to freak people out.

**Comment by rdebath** [comment]
Does this include an interface to the NTP protocol, complete with a wild spin to get to switch correctly to "Summer Time"?

**Comment by klattimer** [comment]
No, in fact the clock functionality is extremely poor. I thought the mechanism inside of the clock would be better, but actually getting the thing to drift accurately is a nightmare.
